manifests/v/valinet/ExplorerPatcher/Prerelease/22000.376.40.12_74b8102/valinet.ExplorerPatcher.Prerelease.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: valinet.ExplorerPatcher.Prerelease
+PackageVersion: 22000.376.40.12_74b8102
```

Version matching needs to be looked at
```suggestion
PackageVersion: 22000.376.40.12
```